hbase-server/src/main/java/org/apache/hadoop/hbase/executor/EventHandler.java

```diff
@@ -152,12 +153,11 @@ public long getSeqid() {
    * priority beyond FIFO, they should override {@link #getPriority()}.
    */
   @Override
-  public int compareTo(Runnable o) {
-    EventHandler eh = (EventHandler)o;
-    if(getPriority() != eh.getPriority()) {
-      return (getPriority() < eh.getPriority()) ? -1 : 1;
+  public int compareTo(EventHandler o) {
+    if(getPriority() != o.getPriority()) {
```

Comment:
nit: include a null check in the beginning for o?

Reply:
Will do